docs/Spark/extensibility/user-defined-functions.md

```diff
@@ -9,46 +9,32 @@ tags:
   - udafs
 ---
 
-Allows you to create user defined functions (UDF) which are then usable anywhere in the Pipeline
+Prophecy lets you create user-defined functions (UDFs) which can be used anywhere in the Pipeline.
 
-### Parameters
+## Parameters
 
-| Parameter               | Description                                                                                                                              | Required |
-| :---------------------- | :--------------------------------------------------------------------------------------------------------------------------------------- | :------- |
-| UDF Name                | Name of the UDF to be used to register it. All calls to the UDF will use this name                                                       | True     |
-| Definition              | Definition of the UDF function. <br/> Eg: `udf((value:Int)=>value*value)`                                                                | True     |
-| UDF initialization code | Code block that contains initialization of entities used by UDFs. This could for example contain any static mapping that a UDF might use | False    |
+| Parameter               | Description                                                                                                                                 | Required |
+| :---------------------- | :------------------------------------------------------------------------------------------------------------------------------------------ | :------- |
+| Function name           | The name of the function as it appears in your project.                                                                                     | True     |
+| UDF Name                | Name of the UDF that will register it. All calls to the UDF will use this name.                                                             | True     |
+| Definition              | Definition of the UDF function. <br/> Eg: `udf((value:Int)=>value*value)`                                                                   | True     |
```

Comment:
I know this is outside the scope of your changes, but let's update "Eg" to "E.g.". Or better yet, use something like "For example".

Reply:
Definitely. I'm pro-`For Example` 100000%